webapp/components/CommentList/CommentList.spec.js

```diff
@@ -26,6 +25,23 @@ describe('CommentList.vue', () => {
               id: 'comment134',
               contentExcerpt: 'this is a comment',
               content: 'this is a comment',
+              author: {
+                id: 'some-user',
+                slug: 'some-slug',
+              },
+            },
+            {
+              id: 'comment135',
+              contentExcerpt: 'this is a deleted comment',
+              content: 'this is a deleted comment',
+              deleted: true,
+              author: { id: 'some-user' },
+            },
```

Comment:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-08-04T13:15:42Z" title="Tuesday, August 4th 2020, 3:15:42 pm +02:00">Aug 4, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -28,6 +28,13 @@ describe('CommentList.vue', () => {
               content: 'this is a comment',
               author: { id: 'some-user' },
             },
+            {
+              id: 'comment135',
+              contentExcerpt: 'this is a deleted comment',
+              content: 'this is a deleted comment',
+              deleted: true,
+              author: { id: 'some-user' },
+            },
```


Because of a different solution I suggest in the next comment👇🏼 this test makes no sense anymore.

Would be cool to replace it with a test in `backend/src/schema/resolvers/posts.spec.js` which checks that only comments not deleted and not disabled are counted by the Cypher queries.

Reply:
> [<img alt="Elweyn" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Elweyn) **Authored by [Elweyn](https://github.com/Elweyn)**
_<time datetime="2020-08-05T07:56:58Z" title="Wednesday, August 5th 2020, 9:56:58 am +02:00">Aug 5, 2020</time>_

---


Thanks for the response. I think it is important to check if their are deleted comments that it does not break the other tests, so we could let this one and add another one in backend/src/schema/resolvers/posts.spec.js to check if the count is setten correctly what do you think?